sql/updates/world/master/2017_11_26_00_world.sql

```diff
@@ -0,0 +1,59 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET NAMES utf8 */;
+/*!50503 SET NAMES utf8mb4 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+
+DROP TABLE IF EXISTS `creature_template_resistance`;
+CREATE TABLE IF NOT EXISTS `creature_template_resistance` (
+  `CreatureID` mediumint(8) unsigned NOT NULL,
+  `Resistance1` smallint(6) DEFAULT NULL,
+  `Resistance2` smallint(6) DEFAULT NULL,
+  `Resistance3` smallint(6) DEFAULT NULL,
+  `Resistance4` smallint(6) DEFAULT NULL,
+  `Resistance5` smallint(6) DEFAULT NULL,
+  `Resistance6` smallint(6) DEFAULT NULL,
+  `VerifiedBuild` smallint(5) DEFAULT '0',
+  PRIMARY KEY (`CreatureID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `creature_template_spell`;
+CREATE TABLE IF NOT EXISTS `creature_template_spell` (
```

Comment:
Would be cleaner to have a table "CreatureID, index, spell", most creature have 1 or 2 spells max, no need of all those columns here

We still need index because some script (TotemAI for example) care about spell order

Reply:
i agree.